semgrep/semgrep/rule_schema.yaml

```diff
@@ -190,6 +190,8 @@ definitions:
             type: string
           regex:
             type: string
+          constant-propagation:
```

Comment:
Remember to copy this rule schema to Semgrep-app, it seems that they are still using their own copy.